cmd/vicadmin/server.go

```diff
@@ -120,6 +115,7 @@ func (s *server) serve() error {
 
 	s.mux.Handle("/css/", http.StripPrefix("/css/", http.FileServer(http.Dir("css/"))))
 	s.mux.Handle("/images/", http.StripPrefix("/images/", http.FileServer(http.Dir("images/"))))
+	s.mux.Handle("/fonts/", http.StripPrefix("/fonts/", http.FileServer(http.Dir("fonts/"))))
```

Comment:
Yeah, we do.  If the user doesn't have already have the fontawesome fonts installed locally, then they will get colored boxes instead of the desired icons